temi_code/curate_my_heart_counts_public_BRIDGE.R

```diff
@@ -212,33 +302,31 @@ demographics_tz_from_zip_prefix <- function(demographics_synId) {
     group_by(zip3) %>%
     slice(which.max(n)) %>%
     select(zip3, state)
+  zip_state_uk <- 
```

Comment:
Are you missing some code here? It doesn't look like this would run

Reply:
just updated this!